jsx/Breadcrumbs.js

```diff
@@ -10,11 +10,13 @@
  *
  */
 
+import React, {Component} from 'react';
```

Comment:
@xlecours had mentioned that there might be a way to do this globally so that we are able to import `React`, `Component` and `PropTypes` to all `jsx` files. What are your thoughts Xavier?

Reply:
@HenriRabalais @xlecours Stackoverflow has an answer on how to do it found here:
https://stackoverflow.com/questions/30560883/es6-global-import
Yet, I'm not necessarily sure if it's good practice to globally import modules instead of just writing the imports necessary to the component. Also the way .eslintrc.json is configured in major will give errors if something isn't used in a component. Such as a component that doesn't use PropTypes. Thoughts?

Reply:
Sounds good to me! @xlecours ?